eleventy.config.cjs

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const formatDistanceToNow = require('date-fns/formatDistanceToNow');
+const yaml = require('js-yaml');
+const markdown = require('markdown-it');
+const markdownDefList = require('markdown-it-deflist');
+const typogrify = require('typogr');
+const { EleventyRenderPlugin } = require('@11ty/eleventy');
+
+/** @param {import('@11ty/eleventy').UserConfig} eleventyConfig */
+module.exports = (eleventyConfig) => {
+  eleventyConfig.addPassthroughCopy('source/assets/dist');
+  eleventyConfig.addPassthroughCopy('source/assets/img');
+  eleventyConfig.addPassthroughCopy('source/favicon.ico');
+
+  eleventyConfig.setLiquidOptions({
+    jsTruthy: true,
+  });
+  eleventyConfig.setUseGitIgnore(false);
+
+  const mdown = markdown({
+    html: true,
+    typographer: true,
+  }).use(markdownDefList);
```

Comment:
Do we need to turn off indented code blocks here?

Reply:
They're disabled by default in the latest versions of 11ty: https://www.11ty.dev/docs/languages/markdown/#indented-code-blocks